### PR TITLE
[Hub Generated] Review request for Microsoft.Devices to add version 2019-03-12

### PR DESCRIPTION
**Pull request by jlian**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/app/branch/jlian/azure-rest-api-specs/dev-iothub-Microsoft.Devices-ops-mon-stable).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5478 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5478/commits/0c60a919da00031c0b3a2ef8a4bbb775c07c85a7

File: specification/iothub/resource-manager/Microsoft.Devices/stable/2019-03-12/examples/iothub_delete.json
Comment (dsgouda) [inline specification/iothub/resource-manager/Microsoft.Devices/stable/2019-03-12/examples/iothub_delete.json]: nit: its good practice to avoid using production style subscription ids

**Review by dsgouda — CHANGES_REQUESTED** [review]

**Comment by dsgouda** [comment]
Looks good for the most part, just need a quick approval from ARM for the deleted models

**Comment by jlian** [comment]
Thanks. This means we have to meet with ARM team right?

**Comment by dsgouda** [comment]
@jlian ARM will post their comments on this PR, don't think you need to meet them since it is a relatively small change

**Comment by jlian** [comment]
Ok thanks. We're not ready for this to be merged into master yet (I'm preempting a little with this PR). Just want to make sure that we can get a heads up before the merge? Or is there a "hold-off" label like docs we can use?

**Comment by dsgouda** [comment]
I have added the `DoNotMerge` label

**Comment by dsgouda** [comment]
startbuild

**Comment by dsgouda** [comment]
startbuild

**Review by KrisBash — APPROVED** [review]
LGTM

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
@jlian We are ready to merge this one. Please let us know when we can.

**Comment by jlian** [comment]
We're still getting ready on our side. Sorry for the delay

**Comment by dsgouda** [comment]
@jlian gentle reminder, we will close the PR by EOW if there is no activity

**Comment by dsgouda** [comment]
startbuild

**Comment by jlian** [comment]
Gonna close this until we can figure it out form our end

**Pull request closed without merging**
